From: Tobias Richter <Tobias.Richter@diamond.ac.uk>
Date: Wed, 30 Jun 2010 10:17:38 +0100
Subject: look for sonamed version of library

---
 bindings/python/nxs/napi.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bindings/python/nxs/napi.py b/bindings/python/nxs/napi.py
index 6c205f8..e1138ad 100755
--- a/bindings/python/nxs/napi.py
+++ b/bindings/python/nxs/napi.py
@@ -234,7 +234,7 @@ def _libnexus():
             lib = 'libNeXus.dylib'
             ldenv = 'DYLD_LIBRARY_PATH'
         else:
-            lib = 'libNeXus.so'
+            lib = 'libNeXus.so.0'
             ldenv = 'LD_LIBRARY_PATH'
         # Search the load library path as well as the standard locations
         ldpath = [p for p in os.environ.get(ldenv,'').split(':') if p != '']
-- 
